docs/read_the_docs/source/workflow_inputs/characteristics.rst

```diff
@@ -1,6 +1,6 @@
 .. _housing_characteristics:
 
-Housing Characteristics
+Housing Characteristics Distributions
```

Comment:
I think you need to make this change instead in `docs/read_the_docs/source/workflow_inputs/create_characteristics_rst.rb` -- this script automatically generates characteristics.rst.